common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -484,8 +472,9 @@ def _get_render_metadata(self, context, display_items, prereq_met, prereq_meta_i
             'sequence_name': self.display_name,
             'exclude_units': context.get('exclude_units', False)
         }
-        if self.gated_sequence_fragment:
-            params['gated_sequence_fragment'] = self.gated_sequence_fragment.content
+
+        if self.gated_sequence_paywall:
+            params['gated_sequence_paywall'] = self.gated_sequence_paywall
```

Comment:
Is there a downside to having this parameter exist even if the value is None. Wondering if we can just add this into the params above directly.